doc/install.rst

```diff
@@ -110,6 +112,25 @@ Anaconda offers scikit-learn as part of its free distribution.
     via ``pip install``.
 
 
+Intel conda channel
+-------------------
+
+Intel maintains a dedicated conda channel that ships scikit-learn::
+
+    conda install -c intel scikit-learn
```

Comment:
Does this mean that "conda install scikit-learn" installs a different codebase when pulling from the conda channel?

This would be a bit problematic from a point of view of communication with the community. Basically, it would mean that a package called "scikit-learn" is not the scikit-learn codebase.

Reply:
The scikit-learn code is the same it's just the __init__ of scikit-learn that is changed to trigger the monkeypatching from daal4py (with a user notification).

Reply:
On PyPI, the package is called "intel-scikit-learn" (https://pypi.org/project/intel-scikit-learn/#description), I think that it would be clearer if the same was done for conda, so you would do `conda install intel-scikit-learn`

Reply:
That would be a good solution. Is there a way in conda to make sure that scikit-learn is not installed when intel-scikit-learn is installed? What about projects with a dependency on scikit-learn?

Reply:
I agree that this is very confusing and will result in headaches on our side and probably the user side.

Reply:
Indeed this behavior sounds extremely surprising and headache inducing. Changing a conda channel should not provide a version of the code that was never a scikit-learn release, in my opinion.

Reply:
An update on this discussion: Intel has changed to disable monkey-patching by default. In other terms: when installing scikit-learn from the Intel channel, unmodified scikit-learn code is run by default and the user needs to take a specific action. Hence I think that:

* We can safely point to the Intel channel
* We should explain how it works briefly here

@oleksandr-pavlyk, is there a webpage that we can point to (for instance a README somewhere) to describe this process. It might be useful for users.